src/main/java/com/prgrms/commandLineApplication/customer/validator/CustomerValidator.java

```diff
@@ -0,0 +1,54 @@
+package com.prgrms.commandLineApplication.customer.validator;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.UUID;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class CustomerValidator {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(CustomerValidator.class);
+
+  private static final String EMAIL_FORMAT = "^[_a-z0-9-]+(.[_a-z0-9-]+)*@(?:\\w+\\.)+\\w+$";
+
+  private static final String INVALID_ID_ERROR = "Invalid ID";
+  private static final String INVALID_EMAIL_ERROR = "Invalid Email";
+
+  public static void checkId(UUID id) {
+    if (id == null) {
+      LOGGER.error("Customer Error Message => {}", INVALID_ID_ERROR);
+      throw new IllegalArgumentException(INVALID_ID_ERROR);
+    }
+  }
+
+  public static void checkEmail(String email) {
+    if (isEmailNull(email) || !isValidEmailFormat(email)) {
```

Comment:
```suggestion
    if (email == null || !isValidEmailFormat(email)) {
```
이렇게만 작성해도 가독성면에서는 문제없을 것 같아요

Reply:
31dcdfd25e0d1542a1b88f2056b7cbf6b5a4af7c